code/weapon/weapon.h

```diff
@@ -39,7 +39,7 @@ class ship_subsys;
 #define	WP_UNUSED			-1
 #define	WP_LASER			0		// PLEASE NOTE that this flag specifies ballistic primaries as well - Goober5000
 #define	WP_MISSILE			1
-#define	WP_BEAM				2
+#define	WP_BEAM				2		// this is not the main way of distinguishing beams; many beams are actually WP_LASER
```

Comment:
Might add a little bit here pointing them towards the info flag or `is_beam()`, if they're a newcomer and ask the obvious follow-up 'what *is* the main way of distinguishing beams then??'.